src/listeners/messageCreate.ts

```diff
@@ -67,13 +67,41 @@ client.on("messageCreate", async message => {
 
 	// Delete non-stringURL messages in review-strings
 	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
+
 	if (message.channel instanceof TextChannel && message.channel.name.endsWith("-review-strings")) {
-		if (!/(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi.test(message.content))
-			await message.delete()
-		else {
-			await message.react("vote_yes:839262196797669427")
-			await message.react("vote_maybe:839262179416211477")
-			await message.react("vote_no:839262184882044931")
+		if (stringURLRegex.test(message.content)) {
+			const urlsWithComments: string[] = [],
+				contentSplit = message.content.split(" ")
+
+			let commentAtStart = "",
+				encounteredUrl = stringURLRegex.test(contentSplit[0])
+
+			for (const word of contentSplit) {
+				if (stringURLRegex.test(word)) {
+					if (!encounteredUrl) {
+						urlsWithComments.push(`${commentAtStart}${word}`)
+						encounteredUrl = true
+					} else urlsWithComments.push(word)
+				} else if (encounteredUrl) urlsWithComments[urlsWithComments.length - 1] += ` ${word}`
+				else commentAtStart += `${word} `
+			}
+
+			if (urlsWithComments.length === 1) {
+				await message.react("vote_yes:839262196797669427")
+				await message.react("vote_maybe:839262179416211477")
+				await message.react("vote_no:839262184882044931")
```

Comment:
You can avoid a bunch of unnecessary computing if you just match the content with the string URL regex at the very top and if the matched length is 1 run this code, otherwise run the rest